third-party/picocom/impl.h

```diff
@@ -0,0 +1,20 @@
+
+#define _POSIX_PATH_MAX 256
+
+static inline char *ttyname(int fd) {
+	static char buf[16];
+	(void) fd;
+	return strcpy(buf, "/dev/ttyS0");
+}
+
+
+static void cfmakeraw (struct termios *__termios_p) {
```

Comment:
static inline?

Reply:
indeed. I would actually prefer to implement those syscalls before sending the patches. this is just a first crude effort to make it build successfully.

Reply:
OK